pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -633,6 +633,12 @@ func ValidateVirtualMachineInstanceSpec(field *k8sfield.Path, spec *v1.VirtualMa
 					Message: fmt.Sprintf("Masquerade interface only implemented with pod network"),
 					Field:   field.Child("domain", "devices", "interfaces").Index(idx).Child("name").String(),
 				})
+			} else if iface.InterfaceBindingMethod.Bridge != nil && networkData.NetworkSource.Pod != nil && config.GetDefaultNetworkInterface() != "bridge" {
+				causes = append(causes, metav1.StatusCause{
+					Type:    metav1.CauseTypeFieldValueInvalid,
+					Message: fmt.Sprintf("When the deafult network Interface is not bridge, bridge interface is not enabled with pod network"),
```

Comment:
do we need Sprintf here?

Reply:
"default"

Reply:
Hey @fedepaol . 
> do we need Sprintf here?

I don't see a reason why we need Sprintf, but all the other validations in this function were written this way, and I just adopted the same style. do you think we should change that?

Reply:
Sorry for the late reply, I was out. 
It doesn't make any difference but I don't see the point of having `Sprintf` since there is no need to format anything.

Reply:
Thanks. I'll change that